scripts/debian/prerm.in

```diff
@@ -24,10 +24,7 @@ set -e
 case "$1" in
     remove|upgrade|deconfigure)
       echo "[PRE-REMOVE] Stop all Falco services:"
-      systemctl --system stop 'falco-kmod.service' || true
-      systemctl --system stop 'falco-bpf.service' || true
-      systemctl --system stop 'falco-modern-bpf.service' || true
-      systemctl --system stop 'falco-plugin.service' || true
+      systemctl --system stop 'falco.target' || true
```

Comment:
Since all units are `PartOf` falco.target, and falcoctl service is bound to it, we can just stop `falco.target`.